app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{" + f"Name: {self.name}, " \
+                     f"Health: {self.health}, " \
+                     f"Hidden: {self.hidden}" + "}"
+
+
+class Carnivore(Animal):
+
+    def bite(self, animal: Animal) -> None:
+        if isinstance(animal, Herbivore) and not animal.hidden:
+            animal.health -= 50
+        if animal.health <= 0:
+            self.alive.remove(animal)
```

Comment:
I think, u use Class attribute "alive", so u should call it using Class.attribute. 
After that u don't use self in that method, so it will be staticmethod